### DISCUSSION TITLE
Grid View options

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#1520] [General] [answered]
Hi team!
Ridhi and I are currently working on Restricted Access Keys design for RBAC and were hoping to get your feedback on this exploration. Our use case is to show permissions to users for different Twilio products and give them granular control in defining the level of access they'd like for an API key. 
We are considering two different views - List view and a Grid View. Can you please look at List View and Grid View Option 1 designs here and leave feedback either within Abstract or here - whatever works best. We would like to implement the solution using Paste so are interested to hear about your feedback from that perspective. Thank you!

Abstract link: https://share.goabstract.com/f8a3e48a-8ecd-4459-91cd-e66c15997244

**Comment by kate-porter** [comment, marked as answer]
Just wanted to drop a quick summary here of what we discussed via Abstract comments:

Overall, this is looking good! There are some small adjustments that can be made to make the checkbox interactions more accessible by making their behavior more explicit, particularly around the "select all" checkboxes within the list and grid views. We also chatted about renaming the list and grid toggle to instead highlight the mental model difference rather than the format difference. We put some ideas together in this Figma project: https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Feedback?node-id=424%3A27160

For all of those watching at home, there are lots of great discussions sprinkled throughout the Abstract project, too, so take a look!

@pguptaDS - thanks for much for sharing this work with us! I know there is still some iteration happening, so definitely feel free to share this with us again as it evolves, and let us know if you have any other questions or need clarification on any of the feedback. :)

**Reply by pguptaDS** [reply]
Thank you so much @katieporter for helping us think through this. Your detailed figma explorations were super helpful and have given us something to take back to the engineering team. Thank you again :)
